demo/index.html

```diff
@@ -144,7 +144,7 @@
 
         var openWinButton = document.getElementById('open-win');
         openWinButton.addEventListener('click', function() {
-            win = window.open('win.html?x=100&y=100', 'test-window', 'height=100,width=100');
+            win = window.open('win.html?x=100&y=100', 'test-window', 'height=800,width=400');
```

Comment:
since there are no specs for this and since this is a demo page, I think we should be good to go.